### DISCUSSION TITLE
email notification for quota limit

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3431] [Q&A]
Hello Pretix team!

Is there an option to create an email notification, that will triger every time a specific quota of an event is reaching its limit ?
For example every time it reaches the 95% of its quota ?

Thank you in advance

**Comment by raphaelm** [comment]
There is currently no such option. We'd like there to by, but we believe it will be quite hard implementing this properly.

**Comment by alemao8** [comment]
Perhaps you could make use of the new feature added in 4.17 version, the low availability threshold.
You could implement a script where if the threshold is equal to the low availability threshold, then an email notification is sent to a list of users

**Reply by raphaelm** [reply]
Yes, the problem is with handling the notification state, because there is no "event" that triggers when a quota is "low", and we don't want to spam users if it happens multiple times in a row, etc

**Comment by alemao8** [comment]
Could you use a global flag per event that is 0 when a notification is not sent yet and becomes 1 when the notification has been sent ? That way you should send one notification per event and you won't spam the users. I think ..

**Reply by raphaelm** [reply]
Per event? Per quota? Per user? What if you increase the quota later, wouldn't you want to get it again?

There's lots of open design questions here, we currently don't have the capacity to tackle this issue unfortunately.

**Comment by alemao8** [comment]
hmmm yes it is a little more complex ... I see.
